phys/module_bl_ysu.F

```diff
@@ -206,10 +241,30 @@ subroutine ysu(u3d,v3d,th3d,t3d,qv3d,qc3d,qi3d,p3d,p3di,pi3d,               &
                                                                         dvsfc, &
                                                                         dtsfc, &
                                                                         dqsfc
-
+   real,dimension(ims:ime,kms:kme,jms:jme) :: a_u,a_v,a_t,a_e,b_u,b_v,b_t,b_e, &
```

Comment:
Should these be tile-sized, if they are not sent out of the scheme?

Reply:
Sorry for the late reply. I did not understand what "tile-sized" meant. Correct, they are not sent out of the scheme. Let me know any suggested changes. Thank you.

Reply:
@eahendricks Tile size means the dimensions are its:ite, jts:jte and kts:kte. If you look at lines just above yours, you'll see local arrays declared for YSU: rqvbl2dt, qv2d, etc.,

Reply:
@weiwangncar @davegill Thanks for the clarification. As a new WRF user, I probably do not fully understand the differences between memory and tile indices. Do the memory indices include the halo cells for each tile in domain decomp, whereas tile indices do not (so tile size would always be smaller than memory size)? I think you are correct they should be tile sized since they are not sent out. I will have to make the modifications and run it again through my test suite to make sure the results look the same. If so I'll commit the changes.

Reply:
@eahendricks Tile sizes can be smaller than patches which are dimensioned by memory sizes, if another layer of parallelism, OpenMP, is activated. If one just uses dmpar, then tile and patch sizes are the same. The definition for a tile is section of a patch allocated to a shared-memory processor within a node. So the tiles do include halo regions. By changing your array dimensions, it helps to enable shared-memory parallelism to work properly, at least we hope.

Reply:
@weiwangncar Thanks for the explanation. I was using dmpar, which is why I did not notice any problem. So glad you guys caught this error.